deps/ncrypto/engine.cc

```diff
@@ -58,7 +58,7 @@ EnginePointer EnginePointer::getEngineByName(const std::string_view name,
       }
     }
   }
-  return std::move(engine);
+  return engine;
```

Comment:
What's the reason for this particular change?

Reply:
suppress a compile warning about move elision